app/templates/events/view/sessions.hbs

```diff
@@ -1,106 +1,50 @@
-            </div>
-          </td>
-          <td>
-            Test Session
-          </td>
-          <td>
-            <div class="ui ordered list">
-              <div class="item">Mario Behling</div>
-              <div class="item">Test</div>
-            </div>
-          </td>
-          <td>
-            Blockchain
-          </td>
-          <td>
-            Get an outlook of the awesome program of the weekend from track organizers and the content team.
-          </td>
-          <td>
-            March 18, 2017
-            <br> 09:30 AM
-          </td>
-          <td>
-            March 25,
-            <br> 09:30 PM
-          </td>
-          <td>
-            No
-          </td>
-          <td>
-            <div class="ui vertical compact basic buttons">
-              {{#ui-popup content=(t 'View') class='ui icon button'}}
-                <i class="unhide icon"></i>
-              {{/ui-popup}}
-              {{#ui-popup content=(t 'Edit') class='ui icon button'}}
-                <i class="edit icon"></i>
-              {{/ui-popup}}
-              {{#ui-popup content=(t 'Delete') class='ui icon button'}}
-                <i class="trash outline icon"></i>
-              {{/ui-popup}}
-              {{#ui-popup content=(t 'Browse edit history') class='ui icon button'}}
-                <i class="history icon"></i>
-              {{/ui-popup}}
-            </div>
-          </td>
-          <td>
-            <div class="ui vertical compact basic buttons">
-              {{#ui-dropdown class='ui icon bottom right pointing dropdown button'}}
-                <i class="green checkmark icon"></i>
-                <div class="menu">
-                  <div class="item">{{t 'With email'}}</div>
-                  <div class="item">{{t 'Without email'}}</div>
-                </div>
-              {{/ui-dropdown}}
-              {{#ui-dropdown class='ui icon bottom right pointing dropdown button'}}
-                <i class="red remove icon"></i>
-                <div class="menu">
-                  <div class="item">{{t 'With email'}}</div>
-                  <div class="item">{{t 'Without email'}}</div>
-                </div>
-              {{/ui-dropdown}}
-            </div>
-          </td>
-        </tr>
-      </tbody>
-    </table>
+    {{#tabbed-navigation isNonPointing=true}}
+      {{#link-to 'events.view.sessions.index' class='item'}}
+        {{t 'All'}}
+      {{/link-to}}
+      {{#link-to 'events.view.sessions.list' 'pending' class='item'}}
+        {{t 'Pending'}}
+      {{/link-to}}
+      {{#link-to 'events.view.sessions.list' 'accepted' class='item'}}
+        {{t 'Accepted'}}
+      {{/link-to}}
+      {{#link-to 'events.view.sessions.list' 'confirmed' class='item'}}
+        {{t 'Confirmed'}}
+      {{/link-to}}
+      {{#link-to 'events.view.sessions.list' 'rejected' class='item'}}
+        {{t 'Rejected'}}
+      {{/link-to}}
+      <div class="mobile hidden">
```

Comment:
@anu-007 instead of putting everything inside one `sixteen wide column`, make it a row and put the tabbed navigation in one column and the create session, download buttons in another column.